Makefile

```diff
@@ -9,7 +9,13 @@ terraform:
 	$(MAKE) -C terraform
 
 deploy:
+ifeq ($(shell python -c "import wheel as w; \
```

Comment:
This should reside in `common.mk` so as to catch it as early as possible. If doing so breaks the Travis build, we need to update the wheel package on Travis.